trin-core/src/utils/db.rs

```diff
@@ -20,6 +40,12 @@ pub fn get_data_dir(node_id: NodeId) -> String {
         .to_str()
         .unwrap()
         .to_string();
+
+    TRIN_NODE_ID_DATA_DIRS
```

Comment:
Is it possible that once we have multiple subnetworks, the same data dir will be inserted multiple times here? Actually, I think not... but it still might be worth checking that the same data dir cannot be inserted twice? Though I guess that error would be handled anyway by line 73.

Reply:
It is not possible to have the same data dirs here because `TRIN_NODE_ID_DATA_DIRS` is [HashSet](https://doc.rust-lang.org/std/collections/struct.HashSet.html).